test/unit/TicketBroker.js

```diff
@@ -5,17 +5,25 @@ import expectThrow from "../helpers/expectThrow"
 import {expectRevertWithReason} from "../helpers/expectFail"
 import {createTicket, createWinningTicket, getTicketHash} from "../helpers/ticket"
 import {constants} from "../../utils/constants"
+import Fixture from "./helpers/Fixture"
 
 const TicketBroker = artifacts.require("ETHTicketBroker")
 
 contract("TicketBroker", accounts => {
     let broker
+    let fixture
 
     const sender = accounts[0]
     const recipient = accounts[1]
 
+    const unlockPeriod = 20
+
+    before(async () => {
+        fixture = new Fixture(web3)
+    })
+
     beforeEach(async () => {
-        broker = await TicketBroker.new(0)
+        broker = await TicketBroker.new(0, unlockPeriod)
```

Comment:
Should we move this line into the `before` hook and use `Fixture` to reset the state between each test similar to what we do in `test/LivepeerETHTicketBroker.js`?

Reply:
👍